Code/GraphMol/ChemReactions/Wrap/testReactionWrapper.py

```diff
@@ -694,5 +694,21 @@ def testProperties(self):
     self.assertTrue(nrxn.HasProp("intprop"))
     self.assertEquals(nrxn.GetIntProp("intprop"),3)
 
+  def testRoundTripException(self):
```

Comment:
This test doesn't seem related to the PR. Are you sure you meant to include it?
(having more tests is always good, I just want to check)

Reply:
It was related to #1985 like the one in SetQuery, they caused exceptions.  It isn't related to this PR but it isn't a terrible test.  I can remove it if you like

Reply:
Nope, I just wanted to be sure that it didn't accidentally creep in.